src/tools/perf/lib/libperf_int.h

```diff
@@ -225,7 +225,7 @@ static UCS_F_ALWAYS_INLINE void ucx_perf_update(ucx_perf_context_t *perf,
 static UCS_F_ALWAYS_INLINE int
 ucp_perf_is_daemon_mode(const ucx_perf_params_t *params)
 {
-    return (0 != params->ucp.dmn_local_addr.ss_family);
+    return params->ucp.is_daemon_mode;
```

Comment:
can remove the wrapper ucp_perf_is_daemon_mode now

Reply:
sure